pkg/config/config.go

```diff
@@ -83,12 +86,12 @@ func (cfg *Config) BindFlags() {
 	)
 	flag.StringVar(
 		&cfg.AccountID, flagAWSAccountID,
-		"",
+		envutil.WithDefault("AWS_ACCOUNT_ID", ""),
```

Comment:
Please declare `AWS_ACCOUNT_ID` and `AWS_REGION` as constants at the top of this file